### PR TITLE
implements @rstento's suggestions

### PR DESCRIPTION
**Pull request by renatocaval**
Implement changes suggested y @rstento on https://github.com/lagom/lagom-recipes/pull/11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ To download and run these samples you will need:
 1. a [git](https://git-scm.com/downloads) client
 2. [sbt](http://www.scala-sbt.org/download.html) or [Maven](https://maven.apache.org/install.html) depending on the sample.
 3. a [Java Development Kit](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-4. a code editor ([IntelliJ](https://www.jetbrains.com/idea/), [Eclipse](https://www.eclipse.org/downloads/), [Atom](https://atom.io/),... )
+4. a code editor such as ([IntelliJ](https://www.jetbrains.com/idea/), [Eclipse](https://www.eclipse.org/downloads/), [Atom](https://atom.io/),... )
 
-### Using this samples
+### Using these samples
```
Comment (TimMoore) [inline README.md]: @rstento had suggested changing from "samples" to "examples" here, and I agree. I'd prefer to use "examples" consistently throughout the document (and elsewhere). I know that many people are wildly inconsistent about this, but the two words have subtly different meaning.

http://www.differencebetween.net/language/words-language/difference-between-example-and-sample/

> Both “example” and “sample” imply a part and also act like representatives of a whole.
>
> An example, by definition, is a noun that shows and mirrors other things. Examples are used to exemplify and illustrate something. “Example” is also utilized as a tool for the explanation and reinforcement of a particular point. Moreover, examples are used for strict compliance or as a premeditated experience. In this manner, it is expected that the example will be followed and replicated among its audience.
>
> On the other hand, a sample is a small part of something much bigger. Unlike an example, a sample is random and not specific. Samples are often used to describe the quality or nature of a specific whole. “Sample” is often used in statistics or quantitative research as a term to describe part of a target population.

So "example" is appropriate for these recipes, which exist to exemplify and illustrate. "Sample" would be more appropriate if we were picking random code from existing projects... or more usually would be used in the context of something like manufacturing to demonstrate instances of a mass-produced item.

Sorry to nitpick, but while we're in here tweaking things, I thought I'd mention it. 😉

**Comment by ignasi35** [comment]
Just seen this (After I sent https://github.com/lagom/lagom-recipes/pull/15) . May cause conflicts when merging that :-(

**Comment by ignasi35** [comment]
PS: I squashed these commits as this is not a two-step example.

**Comment by renatocaval** [comment]
Oeps! I probably made a mistake when merging @TimMoore PR yesterday night. I did a squash, but it was probably not the goal.

**Pull request merged**